combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -46,31 +49,45 @@ def maybe_append(df1, df2):
 def combine_usafacts_and_jhu(signal, geo, date_range, fetcher=covidcast.signal):
     """Add rows for PR from JHU signals to USA-FACTS signals."""
     print("Fetching usa-facts...")
-    usafacts_df = fetcher("usa-facts", signal, date_range[0], date_range[1], geo)
+    # for hhs and nation, fetch the county data so we can combined JHU and USAFacts before mapping
+    # to the desired geos.
+    geo_to_fetch = "county" if geo in ["county", "hhs", "nation"] else geo
```

Comment:
"county" is not needed in that list. I have a slight impulse to extract to a constant set but it's probably more readable as is.